scalableminds/webknossos
# Avoid proofreading actions while the last one is not finished

Set wk busy during proofreading and also modify the input catchers so that they not only show a loading spinner if wk is busy, but actually block all inputs.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open an annotation with an agglomerate view, activate it and switch to proofreading mode
- Click a couple of agglomerates to load their agglomerate skeletons
- Open the dev tools, switch to the "Network" tab and select "Slow 3G" throttling so that the proofreading action takes longer
- Do a proofreading action by adding or deleting an edge and try to perform another proofreading action in rapid succession
  - The second action should no longer be possible since all inputs to the viewports are blocked and a loading spinner is shown

### Issues:
- fixes #6296 

------
(Please delete unneeded items, merge only when none are left open)
